core/org.openjdk.jmc.flightrecorder.configuration/src/main/java/org/openjdk/jmc/flightrecorder/configuration/IFlightRecorderService.java

```diff
@@ -41,11 +41,10 @@
 import org.openjdk.jmc.common.unit.IConstrainedMap;
 import org.openjdk.jmc.common.unit.IOptionDescriptor;
 import org.openjdk.jmc.common.unit.IQuantity;
-import org.openjdk.jmc.flightrecorder.configuration.IRecorderConfigurationService;
 import org.openjdk.jmc.flightrecorder.configuration.events.EventOptionID;
 import org.openjdk.jmc.flightrecorder.configuration.events.IEventTypeID;
+import org.openjdk.jmc.flightrecorder.configuration.events.IEventTypeInfo;
 import org.openjdk.jmc.flightrecorder.configuration.recording.RecordingOptionsBuilder;
-import org.openjdk.jmc.rjmx.IConnectionHandle;
 
 /**
  * This is the interface for the JDK Flight Recorder controller.
```

Comment:
Line 53 below has a Javadoc error: `error: reference not found
 * {@link IConnectionHandle#getServiceOrNull(Class)}` and there are a few more Javadoc errors in this file.